sampleDriver/src/EvsEnumerator.cpp

```diff
@@ -229,6 +233,14 @@ void EvsEnumerator::enumerateCameras() {
         }
     }
 
+    // Enumerate carla simulation cameras
+    addCaptureDevice("CarlaSimReverse");
```

Comment:
Is it possible to do this by config file to prevent recompiling on changing configuration ?

Reply:
We can make this as a next step. 
Currently config mngr is used for device streaming config